src/explore-education-statistics-admin/src/pages/release/data/__tests__/ReleaseApiDataSetDetailsPage.test.tsx

```diff
@@ -396,10 +399,209 @@ describe('ReleaseApiDataSetDetailsPage', () => {
     ).not.toBeInTheDocument();
   });
 
+  describe('importing', () => {
```

Comment:
If we go with the 'finalise' terminology, would make any relevant text changes to this test suite here and below.